### PR TITLE
Handover memcached

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr Stone Hracek <phracek@redhat.com>

This Pull Request prepares memcached container for handover under sclorg github organization.

- [x] Makefile is used as in httpd24 container
- [x] get rid off dist-gen
- [x] Directory structure as in container httpd24
- [x] Add container-common-scripts

### CODE REVIEW

**Comment by phracek** [comment]
@pkubatrh and @hhorak please have a look on it before handover.

File: Makefile [hunk 1]
```diff
@@ -1,49 +1,14 @@
...
-	docker run -d $(IMAGE_REPOSITORY)
-
-debug: build
-	docker run -t -i $(IMAGE_OPTIONS) -e MEMCACHED_DEBUG_MODE $(IMAGE_REPOSITORY) bash
-
-build: doc
-	docker build --tag=$(IMAGE_REPOSITORY) -f Dockerfile.rendered .
+include common/common.mk
 
 test: build
```
Comment (pkubatrh) [inline Makefile]: Please remove the `test` and `clean` rules as they are overriding the ones defined in common-container-scripts

**Review by pkubatrh — CHANGES_REQUESTED** [review]
Please also move tests into `1.5/test` and create the files `1.5/.exclude-centos7` and `1.5/.exclude-rhel7`so that we ignore the OS version we do not build the image for.

File: Makefile (hunk 1 above)
Comment (phracek) [reply Makefile]: @pkubatrh All your comments are fixed by the latest commit.

**Comment by pkubatrh** [comment]
lgtm now, thanks!

**Pull request merged**